device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsIotHubConnection.java

```diff
@@ -132,7 +130,7 @@ public void open(Queue<DeviceClientConfig> deviceClientConfigs) throws Transport
                 this.openAsync();
 
                 log.trace("Waiting for authentication links to open...");
-                boolean authenticationSessionOpenTimedOut = !this.authenticationSessionOpenedLatch.await(MAX_WAIT_TO_OPEN_AUTHENTICATION_SESSION, TimeUnit.MILLISECONDS);
+                boolean authenticationSessionOpenTimedOut = !this.authenticationSessionOpenedLatch.await(this.deviceClientConfig.getAmqpOpenAuthenticationSessionTimeout(), TimeUnit.SECONDS);
```

Comment:
FYI - The units have been changed but not the default. Instead of 20000 as default, the value is 20 now. The end result is 20 seconds either way.